### PR TITLE
Russian translation of monitor messages content

### PR DESCRIPTION
**Pull request by Riliane**
Here https://github.com/jenkinsci/jenkins/pull/7272/files among things that appear on the Jenkins config pages, I have translated the admin monitor checkbox texts, but noticed that the actual messages the monitors spawn are yet untranslated.

This is more complicated to see in use, the conditions for the monitor message to show up have got to be satisfied. Some messages are potentially outdated (Java 11, security issues) but even so they could then be a base for a similar change message in the future.

Also some minor updating/formatting of existing line

### Proposed changelog entries

- Add a translation of the administrative monitors' messages to Russian.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [N/A] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [N/A] There is automated testing or an explanation as to why this change has no tests.
- [N/A] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [N/A] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [N/A] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [N/A] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [N/A] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@Pldi23 @yaroslavafenkin @duemir 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7313"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/diagnosis/MemoryUsageMonitor/index_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/hudson/diagnosis/MemoryUsageMonitor/index_ru.properties:24]: Wouldn't `Временной промежуток` make more sense here? (if I get the context right)

File: core/src/main/resources/hudson/model/UpdateCenter/CoreUpdateMonitor/message_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/hudson/model/UpdateCenter/CoreUpdateMonitor/message_ru.properties:29]:
```suggestion
UpgradeProgress=Обновление до Jenkins {0} <a href="{1}">выполняется</a>.
```

File: core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_ru.properties:23]: I really dislike usages of `узел` for nodes even though that seems like a correct translation :D

File: core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties:5]:
```suggestion
  Этот процесс просканирует большую часть директории <code>$JENKINS_HOME</code> (<code>{0}</code>), \
```

**Review by yaroslavafenkin — COMMENTED** [review]
Commented on what has caught my eye at glance. 
I was missing context on some translations, so I'll try to have another look with English version on the side.

File: core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_ru.properties:23]: It was there before me 😊 also, "узел" is used for a node in networking https://ru.wikipedia.org/wiki/%D0%A3%D0%B7%D0%B5%D0%BB_%D1%81%D0%B5%D1%82%D0%B8 and electric circuits https://ru.wikipedia.org/wiki/%D0%A3%D0%B7%D0%B5%D0%BB_%D1%86%D0%B5%D0%BF%D0%B8 , pretty similar connotations.

File: core/src/main/resources/hudson/diagnosis/MemoryUsageMonitor/index_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/diagnosis/MemoryUsageMonitor/index_ru.properties:24]: okay, looking at it again I agree, you pick a timespan within which you get the memory usage stats, so yes, "продолжительность" is too much like "duration"

Comment (Riliane) [reply core/src/main/resources/hudson/diagnosis/MemoryUsageMonitor/index_ru.properties:24]: found "Временной диапазон" in LoadStatistics in the same kind of context, going to unify.

File: core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties:48]: I lack a bit of context here tbh. My best shot is:
`При помощи формы ниже можно привести эти записи к новому формату. Если это сделать, то при откате к старшей версии Jenkins или плагина чтение данных сохраненных в новом формате станет невозможным.`

File: core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties:50]: Обратите внимание, что даже обычное использование Jenkins для создания и настройки задач может сохранить данные в новом формате, и сделает их нечитаемыми для старших версий Jenkins или плагинов.

File: core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message_ru.properties:1]: This always implies Java 8, doesn't it? 🤔

File: core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties:7]: `См. этот документ для получения дополнительной информации о последствиях этой операции (или какими могут быть последствия бездействия).`

Fine with tweaking few words here and there if you wish. The message should stay though.

File: core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties:12]:
```suggestion
rekeyInProgress=Выполняется смена ключа. <a href="{0}/log">Вы можете просмотреть лог</a>.
```

File: core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties:7]: Looking at the link and how far in the past it has happened I don't thinking anyone will ever see this translation.

File: core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message_ru.properties:1]: Политика Content-Security-Policy по умолчанию в настоящее время переопределяется с помощью системного свойства <code>hudson.model.DirectoryBrowserSupport.CSP</code>, что является потенциальной проблемой безопасности при просмотре ненадежных файлов.

File: core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties:7]: this is what I mean by:
> Some messages are potentially outdated (Java 11, security issues) but even so they could then be a base for a similar change message in the future.
if there is ever another problem like that that'd require a similar operation this could be used (or if someone hasn't used this Jenkins for a very long time 😆 )

File: core/src/main/resources/hudson/PluginManager/PluginDeprecationMonitor/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/PluginManager/PluginDeprecationMonitor/message_ru.properties:27]: ok, this definitely sounds shorter and smoother, accepting

File: core/src/main/resources/hudson/PluginWrapper/PluginWrapperAdministrativeMonitor/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/PluginWrapper/PluginWrapperAdministrativeMonitor/message_ru.properties:24]: wordy! 😆 but I could take a few words from this. I'll think it over tomorrow

File: core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message_ru.properties:1]: I was trying to avoid very long winding wordy sentences and rather split them, they feel much more wordy in Russian and harder to read somehow. But I think I also can take a few words tomorow

File: core/src/main/resources/hudson/PluginManager/PluginUpdateMonitor/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/PluginManager/PluginUpdateMonitor/message_ru.properties:24]: There's the plugin list after, and a colon before such an enumeration seems natural to me. I probably wouldn't put one if it sounded more like a title - "Плагины, требующие обновления" for example

**Comment by Riliane** [comment]
@yaroslavafenkin I tried to reword things a little partially based on your suggestions, but also trying to keep it not very wordy and long-winded.

File: core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties]: the "чем выбранная" part is because it seems that you can choose a version to which you want to update - which is not necessarily the current, might be earlier? к новому формату does sound better, though.

File: core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties:51]: I don't like that we use "даже" twice here. I'm tentative between keeping as is but dropping `даже если не использовать эту форму` part or rewording to something like
`Обратите внимание, что даже не используя эту форму Jenkins может сохранить данные в новом формате, тем самым сделав их нечитаемыми для старших...`

File: core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties:7]: Reading this creates an impression that there are different ways of performing rekey'ing.
My understanding is that there's single way of doing it, but different implications. 
```suggestion
  Больше информации о различных последствиях выполнения этой операции (или бездействия) \
```

File: core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message_ru.properties
Comment (yaroslavafenkin) [inline core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message_ru.properties:2]: I'd still say "потенциальной проблемой безопасности".

File: core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties:51]: `Обратите внимание, что даже обычное использование Jenkins для создания и настройки задач может сохранить данные и сделать их нечитаемыми для старших версий Jenkins или плагинов без использования эту форму.` maybe?
your latter option kind of drops the "dangerous" actions specified - job creation and config. I think it's something worth keeping

File: core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties:7]: `for more implications about different ways of doing this (or not doing this.)`
this sounds indeed like different ways of performing (or not) rekeying. Opening the link, then going to "re-keying" link within, seems that the "different ways" mentioned are "right now in the background on a running Jenkins" vs "scheduled on boot".

File: core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message_ru.properties:2]: would "может быть потенциальной" not be repetitive? I guess "Это может быть проблемой безопасности" vs "Это - потенциальная проблема безопасности"

File: core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage_ru.properties:51]: Fine for me

File: core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties:7]: Then `Больше информации о различных способах и последствиях выполнения этой операции...`

File: core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message_ru.properties:2]: Yeah it is repetitive. I really dislike `может быть` in this context. Can be misinterpreted as `maybe` instead of `can be`. The version I like the most is `является потенциальной проблемой безопасности`

File: core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties:7]: `Because this involves modifying data in $JENKINS_HOME, the administrator should consider the following trade-offs in choosing when to run this.`
from that same page. I feel like the idea of the sentence is "the implications are somewhat different for the different ways to do it", so I still like my version.

File: core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message_ru.properties:1]: I wonder if someone could run it on 9 for example... or it could be parametrized for ease of reuse of the message if one day there's another Java version migration?..

**Review by yaroslavafenkin — APPROVED** [review]

File: core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/message_ru.properties
Comment (Pldi23) [inline core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/message_ru.properties:3]:
```suggestion
actionMessage=Вы можете изменить текущую настройку в разделе Безопасность: {0}.
```

File: core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_ru.properties
Comment (Pldi23) [inline core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_ru.properties:6]: 'Пользователи не пользовались' sounds confusing tbh.

Comment (Riliane) [reply core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_ru.properties:6]: any idea as to how to reword this? "не использовали" doesn't sound any better...

File: core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/message_ru.properties
Comment (Riliane) [reply core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/message_ru.properties]: did that and put it in quotes since it's the name, looks better imo

File: core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_ru.properties
Comment (yaroslavafenkin) [reply core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_ru.properties:6]: Maybe omit "пользователи" completely?
`Часто этими токенами не пользовались` or something in that manner.

Comment (Riliane) [reply core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_ru.properties:6]: "Часто пользователи не нуждались в этих токенах"? "пользователям были не нужны эти токены"?
`Often, users did not use the token, resulting in a larger attack surface than necessary.`

Comment (Riliane) [reply core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_ru.properties:6]: I like Dima's version actually, didn't see it the first time

**Review by Pldi23 — APPROVED** [review]

**Comment by NotMyFault** [comment]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
